en/includes/guides/branding/customize-layouts-with-editor.md

```diff
@@ -0,0 +1,156 @@
+# Customize layouts with editor
+
+{% if product_name == "Asgardeo" %}
+!!! note
+    This feature is only available to customers on the enterprise tier.
+{% endif %}
+
+You can now effortlessly customize the branding and layout of user interfaces such as login, registration, and recovery pages using the new in-console editor.  
+{% if product_name == "WSO2 Identity Server" %}
+This feature enables you to apply changes instantly, eliminating the need to edit files or manually rebuild artifacts.  
+{% else %}
+This feature allows you to make changes instantly, without requiring support from the Asgardeo team.  
+{% endif %}
+
+## Enabling the Editor
+
+To use the editor for customizing layouts, follow these steps:
+
+1. Log in to the {{product_name}} Console.
+2. Navigate to the **Branding** section from the side panel and select the **Styles & Text** subsection.
+3. Enable branding by clicking the **Save & Publish** button in the **Styles & Text** section.
+4. Go to the **Design** tab under the **Styles & Text** section.
+5. In the **Login Layouts** section, select the **Custom** layout card.
+6. Click the **Create** button in the preview section to load the HTML, CSS, and JavaScript editor.
+
+Once you load the editor, you can begin customizing the layout as needed.
+
+## General components of the user interface
+
+All pages in WSO2 Identity Server consist of three primary components, as illustrated below. You can reference these components when creating a custom layout.
+
+![Login Page]({{base_path}}/assets/img/guides/branding/login-page-labelled.png)
+
+## Layout syntax formats
+
+This section describes the special syntax used to define custom layouts.
+
+### Component syntax
+
+The component syntax specifies the placement of general components within the custom layout code. During runtime, the system replaces these syntax elements with actual content.
+
+!!! note
+    To successfully publish a layout, make sure to include the `{% raw %}{{{MainSection}}}{% endraw %}` component in the HTML code. The `{% raw %}{{{ProductHeader}}}{% endraw %}` and `{% raw %}{{{ProductFooter}}}{% endraw %}` components are optional.
+
+```html
+<div class="page-wrapper layout-file">
+    <main class="center-segment">
+        <div class="ui container medium center aligned middle">
+            {% raw %}{{{ProductHeader}}}{% endraw %}
+            {% raw %}{{{MainSection}}}{% endraw %}
+        </div>
+        {% raw %}{{{ProductFooter}}}{% endraw %}
+    </main>
+</div>
+```
+
+### Data Syntax
+
+When customizing layouts using the layout editor, you can leverage `data-*` attributes injected into the `<body>` tag of each page. These attributes allow you to style or control logic dynamically based on the page context.
+
+#### Available `data-*` Attributes
```

Comment:
Please fix this linter error

Reply:
fixed with https://github.com/wso2/docs-is/pull/5371/commits/e865915454aeb95e6b3e99f5a788b235ee4c64a0